qcodes/instrument_drivers/Advantech/PCIE_1751.py

```diff
@@ -0,0 +1,258 @@
+import cffi
+from functools import partial
+import collections
+import os
+import logging
```

Comment:
We use  the convetion of

log = logging.getLogger(__name__)  

to get nicer logggin per module!